enhancements/microshift/microshift-updateability-ostree.md

```diff
@@ -554,7 +503,7 @@ Following section describes scenarios where admin's intervention is needed becau
 
 Depending on MicroShift's health admin might:
 - Unhealthy
-  - Delete MicroShift's data to allow fresh start
+  - Delete MicroShift's data to allow fresh start, or
```

Comment:
is the `or` a mistake or an unfinished sentence?

Reply:
It was meant to be a link between two bullets:

```
- action, or
- alternative action
```

In this case: either delete the data and start fresh or investigate and fix

Reply:
Oh I see, I like the way you explained it. This isn't a big deal, but do you like the idea of putting that wording in the list to make it clearer?

```
- Either, action
- Or, alternative action
```

Reply:
Changed, thanks!